src/graphql/resolvers/employee.ts

```diff
@@ -0,0 +1,243 @@
+import { Query, Resolver, Arg, Mutation } from "type-graphql";
+import Employee, { EmployeeInput } from "../schemas/employee";
+import EmployeeModel from "../../db/models/employee";
+import CarModel from "../../db/models/car";
```

Comment:
I recommend avoiding relative paths when importing things into other files. There are a couple different approaches like https://create-react-app.dev/docs/importing-a-component/#absolute-imports or https://stackoverflow.com/questions/45213279/how-to-avoid-using-relative-path-imports-redux-action-action1-in-cre